src/main/resources/ad-extension.yml

```diff
@@ -1,3 +1,5 @@
 extensionName: ad-extension
 hostAddress: 127.0.0.1
 hostPort: 4532
+opensearchAddress: 127.0.0.1
+opensearchPort: 9200
```

Comment:
This is covered in https://github.com/opensearch-project/anomaly-detection/pull/717

Reply:
It wasn't when I wrote it! :D

Reply:
And #717 has a typo: 127.0.01 ;)